src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/CodeActions/Razor/ComponentAccessibilityCodeActionProvider.cs

```diff
@@ -42,7 +41,7 @@ public async Task<ImmutableArray<RazorVSInternalCodeAction>> ProvideAsync(RazorC
         // We also check for tag helper start tags here, because an invalid start tag with a valid tag helper
         // anywhere in it  would otherwise not match. We rely on the IsTagUnknown method below, to ensure we
         // only offer on actual potential component tags (because it checks for the compiler diagnostic)
-        var startTag = (IStartTagSyntaxNode?)node.FirstAncestorOrSelf<SyntaxNode>(n => n is IStartTagSyntaxNode);
+        var startTag = node.FirstAncestorOrSelf<BaseMarkupStartTagSyntax>();
```

Comment:
Thank you! This is literally the dream I had when I created that interface. If only I'd known how, and was brave enough, to use the syntax generator :)